### PR TITLE
Adding python app to ADB GenAI workshop for cloudworld

### PR DESCRIPTION
**Pull request by martygubar**
WMS ID: 11541

For cloudworld, we are going to use a new python django app to highlight GenAI integration with ADB. This PR provides all of that content.

### CODE REVIEW

**Review by kamryn-v — APPROVED** [review]

**Comment by kamryn-v** [comment]
HI @martygubar,  please resolve your conflicts. Thank you!

File: shared/adb-ai-web-app/openapi-integration/images/2024-08-06_21-46-27.png
Comment (hope-fisher) [inline shared/adb-ai-web-app/openapi-integration/images/2024-08-06_21-46-27.png:1]: This image needs a descriptive name

File: movie-stream-story-lite/apply-graph/files/Movie Recommendations - Personalized SALSA.dsnb
Comment (hope-fisher) [inline movie-stream-story-lite/apply-graph/files/Movie Recommendations - Personalized SALSA.dsnb:1]: This file name needs to be all lowercase.

**Comment by hope-fisher** [comment]
Hey @martygubar I added a couple comments for small things, but I also want to ensure the files included in these changes are correct. The WMS entry points to the adb-ai-web-app folder, but this PR includes adb-speaks-human and movie-stream-story-lite folders. Can you confirm this would not effect other workshops or is a necessary change to a common file?

Screenshot 2024-08-09 at 1 19 13 PM

File: movie-stream-story-lite/apply-graph/files/Movie Recommendations - Personalized SALSA.dsnb
Comment (martygubar) [reply movie-stream-story-lite/apply-graph/files/Movie Recommendations - Personalized SALSA.dsnb:1]: Sorry. This should not be part of this commit.

**Pull request closed without merging**